packages/ui/package.json

```diff
@@ -7,49 +7,47 @@
   "exports": {
     ".": "./index.ts",
     "./classNames": "./classNames.ts",
-    "./alert": "./components/alert/index.ts",
-    "./app-list-card": "./components/app-list-card/index.ts",
-    "./apps": "./components/apps/index.ts",
-    "./arrow-button": "./components/arrow-button/index.ts",
-    "./avatar": "./components/avatar/index.ts",
-    "./badge": "./components/badge/index.ts",
-    "./breadcrumb": "./components/breadcrumb/index.ts",
-    "./button": "./components/button/index.ts",
-    "./buttonGroup": "./components/buttonGroup/index.ts",
-    "./calendar-switch": "./components/calendar-switch/index.ts",
-    "./card": "./components/card/index.ts",
-    "./command": "./components/command/index.ts",
-    "./createButton": "./components/createButton/index.ts",
-    "./credits": "./components/credits/index.ts",
-    "./dialog": "./components/dialog/index.ts",
-    "./disconnect-calendar-integration": "./components/disconnect-calendar-integration/index.ts",
-    "./divider": "./components/divider/index.ts",
-    "./dropdown": "./components/dropdown/index.ts",
-    "./editable-heading": "./components/editable-heading/index.ts",
-    "./editor": "./components/editor/index.ts",
-    "./empty-screen": "./components/empty-screen/index.ts",
-    "./errorBoundary": "./components/errorBoundary/index.ts",
-    "./form": "./components/form/index.ts",
-    "./form/timezone-select": "./components/form/timezone-select/index.ts",
-    "./head-seo": "./components/head-seo/index.ts",
-    "./icon": "./components/icon/index.ts",
-    "./image-uploader": "./components/image-uploader/index.ts",
-    "./layout": "./components/layout/index.ts",
-    "./list": "./components/list/index.ts",
-    "./logo": "./components/logo/index.ts",
-    "./meta": "./components/meta/index.ts",
-    "./navigation": "./components/navigation/index.ts",
-    "./organization-banner": "./components/organization-banner/index.ts",
-    "./popover": "./components/popover/index.ts",
-    "./scrollable": "./components/scrollable/index.ts",
-    "./sheet": "./components/sheet/index.ts",
-    "./skeleton": "./components/skeleton/index.ts",
-    "./table": "./components/table/index.ts",
-    "./toast": "./components/toast/index.ts",
-    "./tooltip": "./components/tooltip/index.ts",
-    "./top-banner": "./components/top-banner/index.ts",
-    "./filter-select": "./components/filter-select/index.ts",
-    "./unpublished-entity/UnpublishedEntity": "./components/unpublished-entity/index.ts",
+    "./components/alert": "./components/alert/index.ts",
```

Comment:
- If we import from `@calcom/ui/{componentName}` (e.g., `@calcom/ui/badge`), Build goes successful but type-checking fails.
- This is because in tsconfig.json files, we use `"moduleResolution": "Node",`, which does not resolve `exports` object in package.json file for type-checking
- Using `node16` will fix the issue, but let's not do that in this PR because it seems like @zomars tried 2 years ago in https://github.com/calcom/cal.com/commit/e04d81cf11c03d54a9a06285e7ec04b468f3ae75 and reverted